src/main/scala/com/gu/datalakealerts/Features.scala

```diff
@@ -111,6 +111,17 @@ object Features {
 
     override def monitoringQuery(platform: Platform): MonitoringQuery = {
       platform match {
+        case Android =>
+          MonitoringQuery(s"""
+                             |select browser_version, count (distinct page_view_id)
+                             |from clean.pageview
+                             |cross join unnest (component_events) x (c)
+                             |where received_date = date '$yesterday'
+                             |and device_type like '%ANDROID%'
+                             |and c.component.type = 'APP_EPIC'
+                             |and c.action = 'VIEW'
+                             |group by 1
+          """.stripMargin, 124090)
```

Comment:
Braze threshold set as half of yesterday's total impressions, as discussed with @jordanterry